src/mongo.js

```diff
@@ -1,5 +1,9 @@
 // @flow strict
 
+const ERROR_CODES = Object.freeze({
```

Comment:
Given that we're using the latest version of `flow-bin` in this package, how do you feel about giving [enums](https://flow.org/en/docs/enums/) a try?

Reply:
Neat; I had no idea Flow added that, and it looks pretty awesome!

That said, I started going down this path, but then I realized it requires not only additional build, but also an additional runtime dependency to work:
https://flow.org/en/docs/enums/enabling-enums/#toc-enable-enums

Given that it's just a single numeric literal in this tiny tiny package, I feel like it's not worth the overhead to consumers 😄 

So I do want to start using enums, but maybe not in this particular repo